<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PTS Controller - Rb</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PTS</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>f1 initial</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>kHz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>serial port:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PTSf1d">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>f2 initial</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>f3 delta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PTSf1i">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>682.8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>f1 delta</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>kHz</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_PTSf2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>73</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>use?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>loop:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SET_PTS">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>kHz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>kHz</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PTSf3d">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PTSf2d">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>f3 initial</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PTSf3i">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>682.8</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_port_list_PTS">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>kHz</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PTSrefresh">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_PTSf3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>73</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>use?</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_PTS">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>kHz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>f2 delta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PTSf2i">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>682.8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>[NOTE] The SET comment can be used only at trigger = 0 (off).</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Novatech</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_port_list_Nova">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>loop:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>serial port:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Novarefresh">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_Nova">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Port 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>frequency</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>freq change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>phase</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>phase change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>amp change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>amplitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>frequency</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>amplitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>phase change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>amp change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>freq change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>phase</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Port 1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novaf0i">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novaf0d">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NovaA0i">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NovaA0d">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novap0i">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novap0d">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NovaA1i">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novap1i">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novaf1d">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novap1d">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Novaf1i">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NovaA1d">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>20</width>
        <height>291</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SET_Nova">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>amp: 0~1</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chung-You Shih @ lab107 IAMS</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SET ALL</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>PTS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>85</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Notatech</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
